src/client/serialize/SConscript

```diff
@@ -0,0 +1,32 @@
+"""Build FUSE client"""
+import daos_build
+
+def scons():
+    """Execute build"""
+    Import('env')
+
+    denv = env.Clone()
+
+    have_hdf5 = True
+
+    """Do hdf_serialize configure checks"""
+    if GetOption('help') or GetOption('clean'):
+        return denv
+    #If hdf5 installed build hdf_serialize library
+    conf = Configure(denv)
+    if not conf.CheckLibWithHeader('hdf5', 'hdf5.h', 'c'):
+        have_hdf5 = False
+    denv = conf.Finish()
+
+    libraries = ['daos', 'hdf5']
+
+    src = ['hdf_serialize.c']
+    if have_hdf5 is True:
+        print("Building with hdf5 bindings.")
+    	hdf_serialize = daos_build.library(denv, 'hdf_serialize', src, LIBS=libraries)
```

Comment:
(pylint-syntax-error) inconsistent use of tabs and spaces in indentation (<unknown>, line 38)